### PR TITLE
Store bulk customer data in the database

### PR DESCRIPTION
**Pull request by gylaz**
This makes it easier to add and manage new bulk customers.

We can build on this to make it even simpler.

### CODE REVIEW

File: app/models/repo.rb [hunk 1]
```diff
@@ -59,8 +59,9 @@ def stripe_subscription_id
     end
   end
 
-  def exempt?
-    ENV["EXEMPT_ORGS"] && ENV["EXEMPT_ORGS"].split(",").include?(organization)
+  def bulk?
+    BulkCustomer.where(org: organization).any? ||
+      ENV.fetch("EXEMPT_ORGS", "").split(",").include?(organization)
```
Comment (gylaz) [inline app/models/repo.rb]: We can remove this once we move data from `EXEMPT_ORGS` to db.

File: app/models/bulk_customer.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class BulkCustomer < ActiveRecord::Base
+  def self.unpaid
```
Comment (thorncp) [inline app/models/bulk_customer.rb]: What do you think about naming this `unpaid_repos` to be explicit that it returns `Repo` instances?

File: db/migrate/20150725013530_create_bulk_customers.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreateBulkCustomers < ActiveRecord::Migration
+  def change
+    create_table :bulk_customers do |t|
+      t.timestamps null: false
+
+      t.string :org, index: { unique: true }
```
Comment (thorncp) [inline db/migrate/20150725013530_create_bulk_customers.rb]: `null: false` ?

**Comment by thorncp** [comment]
LGTM

**Pull request merged**